src/app/component/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,118 @@
+                      <span
+                        *ngIf="profilePageFormController['email'].errors?.['required'] && (profilePageFormController['email'].dirty || profilePageFormController['email'].touched)"
+                      >
+                        Field is required.
+                      </span>
+                      <span
+                        *ngIf="profilePageFormController['email'].errors?.['email'] && (profilePageFormController['email'].dirty || profilePageFormController['email'].touched)"
+                      >
+                        Email is Invalid.
+                      </span>
+                    </div>
+                  </div>
+                </div>
+                <hr class="my-2 pb-2"/>
+  
+                <div class="row">
+                  <div class="col-sm-3 fw-bold">Phone</div>
+                  <div class="col-sm-9 d-flex justify-content-center flex-column h-100">
+                    <input type="tel" class="form-control text-secondary py-1" formControlName="mobile">
+                    <div class="form-text text-danger fixed-height">
+                      <span
+                        *ngIf="profilePageFormController['mobile'].errors?.['required'] && (profilePageFormController['mobile'].touched || profilePageFormController['mobile'].dirty)"
+                      >
+                        Field is required.
+                      </span>
+                    </div>
+                  </div>
+                </div>
+                <hr class="my-2 pb-2"/>
+  
+                <div class="row">
+                  <div class="col-sm-3 fw-bold">Gender</div>
+                  <div class="col-sm-9 d-flex justify-content-center flex-column h-100">
+                    <input type="text" class="form-control text-secondary py-1" formControlName="gender">
+                    <div class="form-text text-danger fixed-height">
+                      <span
+                        *ngIf="profilePageFormController['gender'].errors?.['required'] && (profilePageFormController['gender'].touched || profilePageFormController['gender'].dirty)"
+                      >
+                        Field is required.
+                      </span>
+                    </div>
+                  </div>
+                </div>
+                <hr class="my-2 pb-2" />
+  
+                <div class="row">
+                  <div class="col-sm-3 fw-bold">Date of Birth</div>
+                  <!-- <div class="col-sm-9 d-flex justify-content-center flex-column h-100">
+                    <input type="text" class="form-control text-secondary py-1" formControlName="dateOfBirth">
+                     <div class="form-text text-danger fixed-height">
+                      <span
+                        *ngIf="profilePageFormController['dateOfBirth'].errors?.['required'] && (profilePageFormController['dateOfBirth'].touched || profilePageFormController['dateOfBirth'].dirty)"
+                      >
```

Comment:
remove commentted code

Reply:
this code is usable so i have updated it.